src/commands/app/init.js

```diff
@@ -250,7 +250,10 @@ class InitCommand extends TemplatesCommand {
 
     switch (selection) {
       case 'orgTemplates': {
-        const supportedServiceCodes = new Set(orgSupportedServices.map(s => `|${s.code}`)) // | symbol denotes an OR clause
+        const supportedServiceCodes = new Set(orgSupportedServices.map((elem, index) => {
+          const operator = index > 0 ? '|' : '' // | symbol denotes an OR clause (only if it's not the first item)
+          return `${operator}${elem.code}`
```

Comment:
Should we add tests to verify this formatted o/p?

Reply:
Good idea, will add

Reply:
Done